README.md

```diff
@@ -1,2 +1,61 @@
-# to-do-list-code-review
-Code review exercise
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# TO-DO-LIST
+
+> To-do lists offer a way to increase productivity, stopping you from forgetting things, helps prioritise tasks, manage tasks effectively, use time wisely and improve time management as well as workflow.
+
+## Built With
+
+- HTML
+- Javascript
+- CSS
+
+To get a local copy up and running follow these simple example steps.
+
+1. Clone Repository
+   ```
+   git clone git@github.com:https://github.com/braincee/to-do-list-package.git
```

Comment:
please use the correct link to your repository. to-do-list-package.git is not the correct repository. 😍